### PR TITLE
Remove unused cert

### PR DESCRIPTION
**Pull request by bgavrilMS**
Test only

**Changes proposed in this request**

**Testing**

**Performance impact**

### CODE REVIEW

File: tests/Microsoft.Identity.Test.Common/TestConstants.cs [hunk 1]
```diff
@@ -32,7 +32,6 @@ public static HashSet<string> s_scope
         public const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
         public const string ClientCredentialAudience = "https://login.microsoftonline.com/72f988bf-86f1-41af-91ab-2d7cd011db47/v2.0";
         public const string AutomationTestThumbprint = "378938210C976692D7F523B8C4FFBB645D17CE92";
-        public const string RSATestCertThumbprint = "A7F68F87199F412451BF0FB364A3358A949D2CA8";
```
Comment (bgavrilMS) [inline tests/Microsoft.Identity.Test.Common/TestConstants.cs:35]: @trwalke - do you remember where this was used?

Comment (trwalke) [reply tests/Microsoft.Identity.Test.Common/TestConstants.cs:35]: it is here. I think this cert is used for our lab access
https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/asset/Microsoft_Azure_KeyVault/Certificate/https://buildautomation.vault.azure.net/certificates/RSATestCertDotNet

Comment (bgavrilMS) [reply tests/Microsoft.Identity.Test.Common/TestConstants.cs:35]: Looks like the tests have all passed though. We probably do not need it in code.

**Review by trwalke — APPROVED** [review]

File: tests/Microsoft.Identity.Test.LabInfrastructure/CertificateHelper.cs [hunk 2]
```diff
@@ -52,5 +52,7 @@ public static X509Certificate2 FindCertificateByThumbprint(string thumbprint, St
 
             }
         }
+
+      
     }
```
Comment (trwalke) [inline tests/Microsoft.Identity.Test.LabInfrastructure/CertificateHelper.cs:57]: extra spaces

**Review by gladjohn — APPROVED** [review]
Looks good.

File: tests/Microsoft.Identity.Test.Common/TestConstants.cs (hunk 1 above)
Comment (gladjohn) [reply tests/Microsoft.Identity.Test.Common/TestConstants.cs:35]: Updated LabAccessThumbPrint with AutomationTestThumbprint value and Lab access is working as expected, and all tests are passing

**Review by pmaytak — APPROVED** [review]

**Pull request merged**